crates/tauri/src/event/mod.rs

```diff
@@ -124,20 +111,20 @@ impl FromStr for EventTarget {
 #[derive(Clone)]
 pub struct EmitArgs {
   /// Raw event name.
-  pub event_name: String,
+  pub event_name: EventName<String>,
```

Comment:
let's remove this field and use the `event` field, we don't need both, see https://github.com/tauri-apps/tauri/pull/12460#discussion_r1923889703